scripts/maintenance/setProofRequest.ts

```diff
@@ -3,17 +3,20 @@ import { packV3ValidatorParams, packValidatorParams } from "../../test/utils/val
 import { calculateQueryHashV2, calculateQueryHashV3 } from "../../test/utils/query-hash-utils";
 import { Blockchain, DidMethod, NetworkId, DID } from "@iden3/js-iden3-core";
 import { buildVerifierId } from "../deploy/deployCrossChainVerifierWithRequests";
-import { Operators } from "@0xpolygonid/js-sdk";
-import {
-  contractsInfo,
-  CIRCUIT_ID_MTP_V2,
-  CIRCUIT_ID_SIG_V2,
-  CIRCUIT_ID_V3,
-} from "../../helpers/constants";
+import { byteEncoder, CircuitId, Operators } from "@0xpolygonid/js-sdk";
```

Comment:
Why CircuitId from js-sdk is better than the same from local constants?

Reply:
Because It's easier to maitain circuit names. For example when V3 circuits will be released, we need remove `-beta` suffix
So I guess better to do it in one place

Reply:
Maybe then we could replace everywhere in `contracts` repo where we use constants like CIRCUIT_ID_MTP_V2, CIRCUIT_ID_SIG_V2, CIRCUIT_ID_V3 and replace them with js-sdk constants?

Reply:
fixed